Modules/xSharePoint/DSCResources/MSFT_xSPCacheAccounts/MSFT_xSPCacheAccounts.psm1

```diff
@@ -4,9 +4,10 @@ function Get-TargetResource
     [OutputType([System.Collections.Hashtable])]
     param
     (
-        [parameter(Mandatory = $true)] [System.String] $WebAppUrl,
-        [parameter(Mandatory = $true)] [System.String] $SuperUserAlias,
-        [parameter(Mandatory = $true)] [System.String] $SuperReaderAlias,
+        [parameter(Mandatory = $true)]  [System.String]  $WebAppUrl,
+        [parameter(Mandatory = $true)]  [System.String]  $SuperUserAlias,
+        [parameter(Mandatory = $true)]  [System.String]  $SuperReaderAlias,
+        [parameter(Mandatory = $false)] [System.Boolean] $SetWebAppPolicy = $true,
```

Comment:
Do you really want to default this parameter to True? I would suggest to mark it as False, so it is a conscious decision to change the web app policy.